kalite/distributed/settings.py

```diff
@@ -37,6 +36,7 @@ def USER_FACING_PORT():
 # TODO(bcipolli): change these to "login" and "logout", respectively, if/when
 #  we migrate to a newer version of Django.  Older versions require these
 #  to be set if using the login_required decorator.
+# TODO(benjaoming): Use reverse_lazy for this sort of stuff
 LOGIN_URL = "/securesync/login/"
 LOGOUT_URL = "/securesync/logout/"
```

Comment:
No, but I was unsure of their exact function as settings and didn't know
how to migrate them.

On Mon, 4 May 2015 13:59 Michael Gallaspy <notifications@github.com> wrote:

> In kalite/distributed/settings.py
> <https://github.com/learningequality/ka-lite/pull/3591#discussion_r29623169>
> :
>
> > @@ -37,6 +36,7 @@ def USER_FACING_PORT():
> >  # TODO(bcipolli): change these to "login" and "logout", respectively, if/when
> >  #  we migrate to a newer version of Django.  Older versions require these
> >  #  to be set if using the login_required decorator.
> > +# TODO(benjaoming): Use reverse_lazy for this sort of stuff
> >  LOGIN_URL = "/securesync/login/"
> >  LOGOUT_URL = "/securesync/logout/"
>
> @rtibbles <https://github.com/rtibbles> I don't think these are valid
> URLs anymore, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/learningequality/ka-lite/pull/3591/files#r29623169>.
>